tests/postgres_tests/test_ranges.py

```diff
@@ -304,6 +304,26 @@ def test_contains_range(self):
             [self.objs[0]],
         )
 
+    def test_decimal_contains_range(self):
+        decimals = RangesModel.objects.bulk_create(
+            [
+                RangesModel(decimals=NumericRange(None, 10)),
+                RangesModel(decimals=NumericRange(10, None)),
+                RangesModel(decimals=NumericRange(5, 15)),
+                RangesModel(decimals=NumericRange(5, 15, "(]")),
+            ]
+        )
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(decimals__contains=199), [decimals[1]]
+        )
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(decimals__contains=1), [decimals[0]]
+        )
+        self.assertSequenceEqual(
+            RangesModel.objects.filter(decimals__contains=15),
+            [decimals[1], decimals[3]],
+        )
```

Comment:
```suggestion
        for contains, objs in [
            (199, [decimals[1]]),
            (1, [decimals[0]]),
            (15, [decimals[1], decimals[3]]),
        ]:
            with self.subTest(decimal_contains=contains):
                self.assertSequenceEqual(
                    RangesModel.objects.filter(decimals__contains=contains), objs
                )
```
can use subtest here :+1: